react/features/settings/components/native/styles.js

```diff
@@ -64,6 +64,11 @@ export default createStyleSheet({
         padding: 5
     },
 
+    settingsForm: {
```

Comment:
Since we defined the abstract Modal to be transparent, we need to add a white background to the Settings modal, as that's the only exception where we don't expect any level of transparency.